### ISSUE TITLE
[accessibility] `removeOutsideVisibleBounds` is problematic for keyboard-only and screen reader users

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet.markercluster#1058]
`removeOutsideVisibleBounds` is problematic for screen reader users, https://github.com/Leaflet/Leaflet/issues/3472#issuecomment-794280789:

> simply hiding out-of-view markers is problematic to non-visual users, because (unlike e.g. https://github.com/Esri/a11y-map and https://github.com/mapbox/mapbox-gl-js/issues/10114) there's no feedback to users during/after pan & zoom which means that it would be nearly impossible for these users to reach/discover the initially hidden markers.

For keyboard-only users it's also very hard to reach or even discover any markers that are out of view.

If nothing else, this should be documented. But from an accessibility perspective, it'd be best for `removeOutsideVisibleBounds` to be disabled by default.

**Comment by Malvoz** [comment]
And since marker clustering already is a performance enhancement itself I'd really like to see a consideration for changing the default.
